lib/cortex/snippets/client.rb

```diff
@@ -7,32 +7,6 @@
 module Cortex
   module Snippets
     module Client
-      class << self
```

Comment:
Also it's what's loading in the necessary files like the helper, webpage class, and railtie